UnitTestBot/UTBotJava
# Python fix and refactoring

## Description

* Refactor python-cli
* Add soTimeout for `PythonCoverageReceiver`
* Refactor deprecated code form `PythonDialogProcessor`
* Change log levels and copy java level to python level
* Fix problem with missed imports
* Filter doctests from collected constants
* Refactor execution timeout handler

## How to test

### Manual tests

* Check matrix from TheAlgorithms (especially from cli)
* Check rendering of `collections.Counter`
* Check changing loglevel when you run cli with --verbosity DEBUG
* Check code with doctests: no one test with doctests as constants

## Self-check list

Check off the item if the statement is true. Hint: [x] is a marked item.

Please do not delete the list or its items.
